dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -949,25 +949,18 @@
                     <p class="text-warning">
                         <span class="glyphicon glyphicon-warning-sign"/> #{bundle['dataset.rejectMessage']}
```

Comment:
Can you go through with labels from the task description and check if everything is ok in this modal?

Reply:
To be honest I didn't realize the task has actual description (thanks @wfenrich ).